test/spec/groupCtrlSpec.js

```diff
@@ -65,3 +65,31 @@ describe('Groups Controller', function() {
     expect(myGuilds).to.be.called
   });
 });
+
+describe("Autocomplete controller", function() {
```

Comment:
Also, I changed $scope to scope, which is not strictly necessary, but it's a stylistic thing for the tests. Not sure why, I think it's just a styling convention that the Angular community has decided on. We don't follow a lot of the conventions, but that is one we do.